webviews/QuickLinksApp.vue

```diff
@@ -97,6 +97,14 @@ onMounted(() => {
           </a>
         </h3>
       </div>
+      <div class="catalogue">
+        <h3>
+          <a href="command:ansible.mcpServer.enabled" title="Activate DevTools MCP Server for AI assistants">
```

Comment:
```suggestion
          <a href="command:ansible.mcpServer.enabled" title="Activate Development Tools MCP Server for AI assistants">
```
For consistency, if we're going to use "Ansible Development Tools" or "Development Tools" to refer to the server